pkg/tinyusb/Kconfig

```diff
@@ -32,6 +32,7 @@ menuconfig PACKAGE_TINYUSB
     select MODULE_TINYUSB_PORTABLE_SYNOPSYS_DWC2 if CPU_STM32 && CPU_FAM_L4
     select MODULE_TINYUSB_PORTABLE_STM32_FSDEV if CPU_STM32 && CPU_FAM_F0
     select MODULE_TINYUSB_PORTABLE_STM32_FSDEV if CPU_STM32 && CPU_FAM_F1
+    select MODULE_TINYUSB_PORTABLE_STM32_FSDEV if CPU_STM32 && CPU_FAM_F3
```

Comment:
Hm, I'm wondering why this wasn't catched by Murdock. Anyway, this selection is buggy and needs to be fixed, which is done by PR #18787.

Reply:
OK, I will drop it then!

Reply:
You can leave it for now, I don't know how long it will take to get PR #18787 merged.